apps/dashboard/src/components/common/SideBar.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useRef, useCallback } from "react";
+import { Moon, Sidebar, Sun, User } from "react-feather";
+import {
+  Button,
+  ButtonSize,
+  ButtonType,
+  Switch,
+  Theme,
+  removeToken,
+  useFormatMessage,
+  useTheme,
+} from "@litespace/luna";
+import { useClosableRef } from "@litespace/luna";
+import cn from "classnames";
+import { useNavigate } from "react-router-dom";
+import { AnimatePresence, motion } from "framer-motion";
+import { useAppDispatch, useAppSelector } from "@/redux/store";
+import { profileSelectors, resetUserProfile } from "@/redux/user/profile";
+import { Route } from "@/lib/route";
+import { SideBarAccordion } from "@/components/common/SideBarAccordion";
+import { NavAccordionItem, NavOption } from "@/types/navbar";
+import SideBarItem from "@/components/common/SideBarItem";
+
+const framerSidebarBackground = {
+  initial: { x: "30rem", y: "0rem" },
+  animate: { x: "0rem", y: "0rem" },
+  exit: { x: "20rem", y: "0rem" },
+  transition: { duration: 0.2 },
+};
+
+const SidebarNav: React.FC<{
+  options: (NavOption | NavAccordionItem)[];
+}> = ({ options }) => {
+  const dispatch = useAppDispatch();
+  const intl = useFormatMessage();
+  const profile = useAppSelector(profileSelectors.user);
+
+  const button = useRef<HTMLDivElement>(null);
+  const { toggle: toggleTheme, theme } = useTheme();
+  const navigate = useNavigate();
+  const {
+    toggle: toggleMenu,
+    open,
+    ref,
+  } = useClosableRef<HTMLUListElement>(button.current);
+
+  const logout = useCallback(() => {
+    removeToken();
+    dispatch(resetUserProfile());
+    navigate(Route.Login);
+  }, [dispatch]);
+
+  return (
+    <nav className="relative">
+      <div
+        ref={button}
+        className="fixed flex flex-col items-center justify-between h-screen p-2 border-l"
+      >
+        <Button
+          onClick={toggleMenu}
+          size={ButtonSize.Small}
+          type={ButtonType.Text}
+          className="!p-2"
+        >
+          <Sidebar className="w-8 h-8" />
```

Comment:
Let's make our icons smaller. Please make all icons `w-6 h-6`. 
```suggestion
          <Sidebar className="w-6 h-6" />
```